Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public string ContentType { get; set; }
+        private HttpWebRequest httpWebRequest { get; set; }
+        private HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper()
+        {
+            ContentType = "application/json";
+        }
+        
+        public string Get(string url, Dictionary<string, string> queryParams = null,
```

Comment:
May be we should add "passing different objects as parameters for request " support?

Reply:
Sorry, don't understand , what do you mean?

Reply:
not rly sure, but I think Nikita talking about Dictionary<string, object>, if i'm right, than answer no